app/views/shelters/pets.html.erb

```diff
@@ -13,7 +13,7 @@
 </header>
 
 <main>
-  <% @pets.each do |pet| %>
+  <% @shelter.sort_pets(params[:adoptable]).each do |pet| %>
```

Comment:
(leaving a note to say we talked about this and will address it with instructors during eval; internet guides suggest params should stay in controller)